validator/accounts/accounts_backup_test.go

```diff
@@ -22,7 +22,7 @@ import (
 	"github.com/prysmaticlabs/prysm/validator/keymanager/derived"
 )
 
-var (
+const (
```

Comment:
The exact same mnemonic repeats in 3 places. I think it warrants moving to a shared package.

Reply:
I thought so as well but didn't come up with any good place. Is there any preference where should I move this?

Reply:
I thought so as well but didn't come up with any good place. Is there any preference where should I move this?

Reply:
I think you can create a new `constants.go` file inside `validator/testing`.